include/service/cache_entity_service.js

```diff
@@ -34,17 +34,23 @@ module.exports = function CacheEntityServiceModule(pb) {
      * @param {String} [objType]
      * @param {String} [valueField]
      * @param {String} [keyField]
+     * @param {String} [site]
+     * @param {String} [onlyThisSite]
      * @param {Integer} The number of seconds that a value will remain in cache 
      * before expiry.
      */
-    function CacheEntityService(objType, valueField, keyField, timeout){
+    function CacheEntityService(objType, valueField, keyField, site, onlyThisSite, timeout){
         this.type       = 'Cache';
         this.objType    = objType;
+        this.valueField = valueField ? valueField : null;
         this.keyField   = keyField;
+        this.site = site || GLOBAL_SITE;
+        this.onlyThisSite = onlyThisSite ? true : false;
         this.timeout    = timeout || 0;
-        this.valueField = valueField ? valueField : null;
     }
```

Comment:
With the addition of the extra parameters we should convert this to a constructor that takes a single options object to prevent the parameters from getting out of control